requirements.txt

```diff
@@ -0,0 +1,13 @@
+Django
```

Comment:
Why use requeriments file is the project is using poetry? Much better to include dependencies in .toml

Reply:
This project is not using poetry at all. pyproject.toml file is a configuration file used, not only for poetry, but also for other tools like `black`. So in this case, requirements file is needed.

Reply:
As @serfer2 said, the pyproject.toml is not only for poetry, you can use it to configure linters, plugins, etc... I'll close this conversation.